libraries/botbuilder-lg/tests/testData/examples/MultiLineExpr.lg

```diff
@@ -0,0 +1,9 @@
+>Demo How LG can evalute a multiple line defined expression
```

Comment:
more kinds of test cases especially corner case
if there is no close bracket. 
```
# template
- ${name
- good
```

```
# template
- ${concat('hi', 
   jack)
```

```
# template
[Activity
  key = ${concat('1',
'2'} | hi
```

more...